docs/sources/auth/auth-proxy.md

```diff
@@ -207,13 +207,13 @@ ProxyPass / http://grafana:3000/
 ProxyPassReverse / http://grafana:3000/
 ```
 
-* Create a htpasswd file. We create a new user **anthony** with the password **password**
+- Create a htpasswd file. We create a new user **anthony** with the password **password**
 
     ```bash
     htpasswd -bc htpasswd anthony password
     ```
 
-* Launch the httpd container using our custom httpd.conf and our htpasswd file. The container will listen on port 80, and we create a link to the **grafana** container so that this container can resolve the hostname **grafana** to the grafana container’s ip address.
+- Launch the httpd container using our custom httpd.conf and our htpasswd file. The container will listen on port 80, and we create a link to the **grafana** container so that this container can resolve the hostname **grafana** to the grafana container’s ip address.
```

Comment:
```suggestion
- Launch the httpd container using our custom httpd.conf and our htpasswd file. The container will listen on port 80, and we create a link to the **grafana** container so that this container can resolve the hostname **grafana** to the Grafana container’s IP address.
```